Makefile

```diff
@@ -48,7 +48,12 @@ lib := \
 
 tcp_rr-objs := tcp_rr_main.o tcp_rr.o rr.o $(lib)
 
-tcp_stream-objs := tcp_stream_main.o tcp_stream.o stream.o $(lib)
+tcp_stream-objs := tcp_stream_main.o tcp_stream.o stream.o tcpdirect.o $(lib)
+
+tcp_stream-cuda-objs := tcp_stream_main_cuda.o tcp_stream.o stream.o tcpdirect.o $(lib)
+
+tcp_stream-cuda2-objs := tcp_stream_main.o tcp_stream.o stream.o tcpdirect.o $(lib)
```

Comment:
clean up so that we only need to build tcp_stream with an additional flag, instead of `make tcp_stream_cuda2`